opencti-platform/opencti-front/src/components/dashboard/WidgetListRelationships.tsx

```diff
@@ -1,189 +1,79 @@
-                          : t_i18n('Restricted')}
-                      </code>
-                    </div>
-                    <div style={bodyItemStyle('10%')}>
-                      <i>
-                        {t_i18n(
-                          `relationship_${stixRelationship.relationship_type}`,
-                        )}
-                      </i>
-                    </div>
-                    <div style={bodyItemStyle('10%')}>
-                      <ItemIcon
-                        type={
-                          stixRelationship.to
-                          && stixRelationship.to.entity_type
-                        }
-                        variant="inline"
-                      />
-                      {/* eslint-disable-next-line no-nested-ternary */}
-                      {stixRelationship.to
-                        ? stixRelationship.to.relationship_type
-                          ? t_i18n(
-                            `relationship_${stixRelationship.to.entity_type}`,
-                          )
-                          : t_i18n(
-                            `entity_${stixRelationship.to.entity_type}`,
-                          )
-                        : t_i18n('Restricted')}
-                    </div>
-                    <div style={bodyItemStyle('18%')}>
-                      <code>
-                        {stixRelationship.to
-                          ? getMainRepresentative(stixRelationship.to)
-                          : t_i18n('Restricted')}
-                      </code>
-                    </div>
-                    <div style={bodyItemStyle('10%')}>
-                      {fsd(stixRelationship[dateAttribute])}
-                    </div>
-                    <div style={bodyItemStyle('12%')}>
-                      {R.pathOr(
-                        '',
-                        ['createdBy', 'name'],
-                        stixRelationship,
-                      )}
-                    </div>
-                    <div style={bodyItemStyle('10%')}>
-                      <ItemMarkings
-                        variant="inList"
-                        markingDefinitions={
-                          stixRelationship.objectMarking ?? []
-                        }
-                        limit={1}
-                      />
-                    </div>
-                  </div>
-                }
-              />
-            </ListItemButton>
-          );
-        })}
-      </List>
+    const iconColumn = {
+      percentWidth: iconWidth,
+      isSortable: false,
+      label: ' ',
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      render: (stixRelationship: any) => (
+        <ItemIcon
+          type={stixRelationship.entity_type}
+          color="primary"
+        />
+      ),
+    };
+    setCurrentColumns({ icon: iconColumn, ...newColumns });
+  }, [columns]);
+
+  if (!currentColumns) return null;
```

Comment:
there is a specific widget in case there are no data, maybe because having no columns shouldn't be possible